test/e2e/common.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
is 1.1 using go 1.17? May need to drop this

Reply:
it should

Reply:
we had some issues a few weeks back with some of the test-infra jobs that were using 1.16 other 1.17.  looks like the pre-submit jobs are using the kubekins are using latest version: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/azure/release-1.20.yaml#L211

Reply:
looking at other files in the 1.1 branch I don't see the 1.17 tags: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/6db87f3116f31eeae4213990ffc9fb0baf356928/test/e2e/azure_logcollector.go#L1

Reply:
can't push updates to this branch since it's owned by the bot? 

```
ERROR: Permission to k8s-infra-cherrypick-robot/cluster-api-provider-azure.git denied to jsturtevant.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```